### PR TITLE
Committing changes to remove verbs from Swagger descriptions

### PR DESCRIPTION
**Pull request by v-shhatt**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -319,11 +319,11 @@
       ],
       "properties": {
         "policySettings": {
-          "description": "Describes policySettings for policy.",
+          "description": "The PolicySettings for policy.",
           "$ref": "#/definitions/PolicySettings"
         },
         "customRules": {
-          "description": "Describes custom rules inside the policy.",
+          "description": "The type of custom rules inside the policy.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Should be just "custom rules", no type

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 2]
```diff
@@ -411,22 +411,22 @@
           "items": {
             "$ref": "#/definitions/OwaspCrsExclusionEntry"
           },
-          "description": "Describes the Exclusions that are applied on the policy."
+          "description": "The type of exclusions that are applied on the policy."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Should be just "exclusions", no type

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 3]
```diff
@@ -411,22 +411,22 @@
...
         },
         "managedRuleSets": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/ManagedRuleSet"
           },
-          "description": "Describes the ruleSets that are associated with the policy."
+          "description": "The type of ruleSets that are associated with the policy."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Should be just "Managed rule sets", no type

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 4]
```diff
@@ -411,22 +411,22 @@
...
         }
       }
     },
     "PolicySettings": {
       "description": "Defines contents of a web application firewall global configuration.",
       "properties": {
         "state": {
-          "description": "Describes if the policy is in enabled state or disabled state.",
+          "description": "The type of state the policy is in enabled or disabled.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]:
```suggestion
          "description": "The state of the policy.",
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 5]
```diff
@@ -438,7 +438,7 @@
           }
         },
         "mode": {
-          "description": "Describes if it is in detection mode or prevention mode at policy level.",
+          "description": "The type of mode if it is in detection or prevention at policy level.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]:
```suggestion
          "description": "The mode of the policy.",
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 6]
```diff
@@ -491,11 +491,11 @@
           "description": "A unique read-only string that changes whenever the resource is updated."
         },
         "priority": {
-          "description": "Describes priority of the rule. Rules with a lower value will be evaluated before rules with a higher value.",
+          "description": "The type of priority of the rule. Rules with a lower value will be evaluated before rules with a higher value.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]:
```suggestion
          "description": "Priority of the rule. Rules with a lower value will be evaluated before rules with a higher value.",
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 7]
```diff
@@ -529,7 +529,7 @@
       }
     },
     "Transform": {
-      "description": "Describes what transforms applied before matching.",
+      "description": "Type of transforms applied before matching.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: No need for "type"

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 8]
```diff
@@ -569,7 +569,7 @@
           }
         },
         "selector": {
-          "description": "Describes field of the matchVariable collection.",
+          "description": "The type of MatchVariable collection.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]:
```suggestion
          "description": "The selector of match variable.",
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json [hunk 9]
```diff
@@ -611,7 +611,7 @@
           }
         },
         "negationConditon": {
-          "description": "Describes if this is negate condition or not.",
+          "description": "Specifies negate condition or not.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: This is still a verb. You can word it as "Whether this is negate condition or not"

**Review by anton-evseev — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json (hunk 2 above)
Comment (v-shhatt) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Done

**Comment by v-shhatt** [comment]
Resolved all the comments

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json (hunk 1 above)
Comment (v-shhatt) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json (hunk 4 above)
Comment (v-shhatt) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json (hunk 5 above)
Comment (v-shhatt) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json (hunk 8 above)
Comment (v-shhatt) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json (hunk 7 above)
Comment (v-shhatt) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/webapplicationfirewall.json]: Done

**Review by anton-evseev — APPROVED** [review]

**Comment by v-shhatt** [comment]
@zikalino : Please merge the PR

Thank you,
Ranjani

**Pull request merged**